### ISSUE TITLE
[Enhancement]: Accessibility: Price ranges not adequately described for screen reader users

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43565]
### Describe the solution you'd like

The current HTML for prices ranges doesn't provide adequate context for screen reader users and can cause confusion. This is the current HTML:

```
<div class="wc-block-grid__product-price price"><span class="woocommerce-Price-amount amount">
     <span class="woocommerce-Price-currencySymbol">$</span>25.00</span> – <span class="woocommerce-Price-amount amount"><span class="woocommerce-Price-currencySymbol">$</span>29.00</span>
</div>
```

Most screen reader users do not have their screen readers set to announce punctuation, so the hyphen is not announced and they hear two amounts announced like "$25 $29." Without the context that this is a price range, it sounds very confusing as to what the price is.

The easiest fix for this is to add an [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) to the price range container like this:

```
<div class="wc-block-grid__product-price price" aria-label="Price range: $25.00 through $29.00"><span class="woocommerce-Price-amount amount">
     <span class="woocommerce-Price-currencySymbol">$</span>25.00</span> – <span class="woocommerce-Price-amount amount"><span class="woocommerce-Price-currencySymbol">$</span>29.00</span>
</div>
```

### Additional context

This should apply to all price ranges through out a site, including in blocks, archive pages, and single product pages.

**Comment by roykho** [comment]
@amberhinds - Thanks for reporting this. We'll take a look when available.

**Comment by amberhinds** [comment]
@vikrampm1 can this be added to your list for next week, please?

**Comment by vikrampm1** [comment]
@amberhinds sure, we will get this tackled as well.
